read-tweet/read.py

```diff
@@ -36,8 +37,11 @@
     card_text = base64.b64encode(card_text.encode("utf-8")).decode("ascii")
 
 tweet_template = Template(tweet)
+name = (
+    f"@{contributor_twitter_username}" if contributor_twitter_username else contributor
```

Comment:
does github return the '@' sign or not?

the example above has it, maybe it was a typo?

Reply:
This documentation [here](https://help.twitter.com/en/managing-your-account/change-twitter-handle) states 
> Your username –– also known as your handle –– begins with the “@” symbol

so it would seem so, but I can't exactly confirm

Reply:
https://docs.github.com/en/rest/reference/users

in the example here, it doesn't seem to be there

`"twitter_username": "monatheoctocat",`

Reply:
just checked here as well: https://docs.github.com/en/graphql/overview/explorer

```graphql
{
  viewer {
    login
    twitterUsername
  }
}
```

and it doesn't have it

```json
{
  "data": {
    "viewer": {
      "login": "marcoacierno",
      "twitterUsername": "marcoacierno"
    }
  }
}
```

also if you add it in the profile settings, it gets removed after saving

Reply:
Yeah when I tested yesterday there wasn't the @...

Reply:
What does the `get-pr-info-action` action return if a user does not have a username?

Reply:
The GitHub API will get `null` (just checked), don't know if it will be turned into a string... Should be a way to test this stuff before doing in production :D, will investigate...